pyaerocom/aeroval/modelmaps_engine.py

```diff
@@ -137,8 +143,20 @@ def _process_map_var(self, model_name, var, reanalyse_existing):
                 logger.info(f"Skipping processing of {outname}: data already exists.")
                 return []
 
-        freq = self.cfg.time_cfg.main_freq
+        mainfreq = self.cfg.time_cfg.main_freq
+        allfreqs = self.cfg.time_cfg.freqs
         tst = TsType(data.ts_type)
+        allfreqs = [TsType(fq) for fq in freqs]  # convert them into TsType objects
```

Comment:
I can not see where `freqs` is defined. Do you meant `allfreqs`?

Reply:
I do